addons/sale_timesheet/views/project_portal_templates.xml

```diff
@@ -2,15 +2,15 @@
 <odoo>
 
     <template id="portal_timesheet_table_inherit" inherit_id="hr_timesheet.portal_timesheet_table">
-        <xpath expr="//thead/tr/th[@t-if='is_uom_day']" position="before">
-	    <t t-set="display_sol" t-value="False"/>
-	    <t t-foreach="timesheets" t-as="timesheet">
-	        <t t-if="timesheet.so_line != task.sale_line_id">
-		    <t t-set="display_sol" t-value="True"/>
-		</t>
-	    </t>
+        <th id="t_label" position="before">
```

Comment:
I would prefer using `name` attribute instead of `id` if you would like to select that item